app/pages/project/index.cjsx

```diff
@@ -115,6 +116,13 @@ ProjectPageController = React.createClass
           # Use apiClient with cached resources from include to get out of cache
           awaitBackground = apiClient.type('backgrounds').get(project.links.background.id).catch((error) => [])
 
+          if project.links?.organization?
+            awaitOrganization = project.get('organization', { listed: true })
+              .catch((error) => [])
+              .then((response) => if response?.display_name then response else null)
+          else
+            awaitOrganization = null
```

Comment:
How about `Promise.resolve(null)` to be consistent with promises being returned in the `Promise.all()` for all of the linked project resources?

Reply:
yes good call, I think I did as intended, but not 100% sure, let me know